src/components/menu/index.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import { useLogoutMutation } from 'services/auth/auth';
+import useTranslation from 'hooks/useTranslation';
+import './styles.scss';
+import icon from 'assets/menu.png';
+
+const Menu = ({ switchTab }) => {
+  const t = useTranslation();
+  const handleLogout = () => logout().then(() => localStorage.removeItem('user'));
+  const [logout] = useLogoutMutation();
+  const changeTab = name => {
+    setIsOpen(!isOpen);
+    switchTab(name);
+  };
+  const [isOpen, setIsOpen] = useState(false);
+  const handleClick = () => {
+    setIsOpen(!isOpen);
+  };
+  return (
```

Comment:
WDYA leaving a blank space between the function declarations and the return statement?